### PR TITLE
Err msg improvement

### PR DESCRIPTION
**Pull request by yogsototh**
slightly improved test too.

### CODE REVIEW

**Review by msprunck — APPROVED** [review]
LGTM!

File: src/ring_jwt_middleware/core.clj [hunk 1]
```diff
@@ -105,12 +105,12 @@
                   (hr-duration (* 1000 before-secs)))
 
           (pos? expired-secs)
-          (format "This JWT has expired since %s (we don't allow JWT older than %s; we only checked creation date and not maximal expiration date)"
+          (format "This JWT has expired %s ago (we don't allow JWT older than %s; we only checked creation date and not maximal expiration date)"
```
Comment (gbuisson) [inline src/ring_jwt_middleware/core.clj]:
```suggestion
          (format "This JWT has expired %s ago (we don't allow JWT older than %s; we only check creation date and not maximal expiration date)"
```

**Review by gbuisson — APPROVED** [review]

File: test/ring_jwt_middleware/core_test.clj [hunk 2]
```diff
@@ -203,35 +203,34 @@
...
-           (sut/validate-jwt "jwt" decoded-jwt-2 86400 test-log-fn))))
-
-  (with-redefs
-    [time/now (constantly (time/date-time 2017 02 16 14 14 11))]
-    (is (= '("This JWT has expired since 1s (we don't allow JWT older than 1 day; we only checked creation date and not maximal expiration date)")
-           (sut/validate-jwt "jwt" decoded-jwt-2 86400 nil test-log-fn)))))
+  (testing "expiration message"
+    (testing "expired time"
+      (let [explain-msg "This JWT has expired %s ago (we don't allow JWT older than 1 day; we only checked creation date and not maximal expiration date)"
+            tst-fn (fn [d expected]
```
Comment (irinarenteria) [inline test/ring_jwt_middleware/core_test.clj:209]: I recommend applying @gbuisson's change suggestion in the messages in the tests too.

**Review by irinarenteria — APPROVED** [review]
LGTM

**Pull request merged**